lib/Backend/Commands.hs

```diff
@@ -91,7 +91,19 @@ createCmd (CreateOptions entryPath _edit force tags fields privateFields) = do
       & E.fields .~ HashMap.fromList allFields
       & E.tags .~ tags
 
-  -- TODO: check that a directory does not exist at the given path.
+    checkForEntriesInEntryPath :: EntryPath -> Sem r ()
+    checkForEntriesInEntryPath entryPath = do
+      let eParentDirs = forM ((NE.tail . Path.entryPathParentDirs) entryPath) Path.pathAsEntryPath
```

Comment:
Suggestion: instead of using `tail`, how about this: we get all parent dirs, and then filter out the root path with `rights`:

```suggestion
    let parentDirsExceptRoot = entryPath
          & Path.entryPathParentDirs 
          & NE.toList
          -- Ignore paths that cannot possibly point to an entry
          <&> Path.pathAsEntryPath
          & rights
```

This way, you don't have to pattern match on `Left/Right` after.

Reply:
Done